docs/en/observability/observability-ai-assistant.asciidoc

```diff
@@ -59,13 +63,52 @@ To set up the AI Assistant:
 [[obs-ai-add-data]]
 == Add data to the AI Assistant knowledge base
 
+[IMPORTANT]
+====
+*If you started using the AI Assistant in technical preview*,
+any knowledge base articles you created before 8.12 will no longer be supported.
```

Comment:
Suggestion: 'will have to be reindexed or upgraded before they can be used'

Reply:
```suggestion
any knowledge base articles you created before 8.12 will have to be reindexed or upgraded before they can be used.
```